src/component/Game.tsx

```diff
@@ -10,7 +10,22 @@ export const Game = () => {
   const [start, setStart] = useState<boolean>(false);
   const [incorrectLetters, setIncorrectLetters] = useState<number>(0);
 
-  const endGame = () => (currentWordIndex < words.length ? false : true);
+  const endGame = () => {
+    if (finalLetter()) {
+      setCurrentWordIndex(currentWordIndex + 1);
+      return true;
+    } else {
+      return currentWordIndex < words.length ? false : true;
```

Comment:
Will this ever evaluate to `true`?

Reply:
@jgayfer The `else` statement does return true for every check after we type the last word. Think about it like after you type the last final letter, you are no longer on the final letter, so we need a check to stop certain components in the next render.

Reply:
Sounds good! We could also kill the `else` here if we wanted, might read a little nicer without the explicit branching.

```
  const endGame = () => {
    if (finalLetter()) {
      setCurrentWordIndex(currentWordIndex + 1);
      return true;
    }

    return currentWordIndex < words.length ? false : true;
  };
```

Reply:
An additional comment I might add.

Should a function that tells us whether or not are in the endgame be manipulating state?